aspnetcore/tutorials/getting-started-with-NSwag.md

```diff
@@ -32,159 +32,194 @@ The main reason to use NSwag is the ability to not only introduce the Swagger UI
 
-# [Visual Studio](#tab/visual-studio)
+### [Visual Studio](#tab/visual-studio)
 
 * From the **Package Manager Console** window:
+  * Go to **View** > **Other Windows** > **Package Manager Console**
+  * Navigate to the directory in which the *TodoApi.csproj* file exists
+  * Execute the following command:
 
     ```powershell
     Install-Package NSwag.AspNetCore
     ```
 
 * From the **Manage NuGet Packages** dialog:
-  * Right-click your project in **Solution Explorer** > **Manage NuGet Packages**
+  * Right-click the project in **Solution Explorer** > **Manage NuGet Packages**
   * Set the **Package source** to "nuget.org"
   * Enter "NSwag.AspNetCore" in the search box
   * Select the "NSwag.AspNetCore" package from the **Browse** tab and click **Install**
 
-# [Visual Studio for Mac](#tab/visual-studio-mac)
+### [Visual Studio for Mac](#tab/visual-studio-mac)
 
 * Right-click the *Packages* folder in **Solution Pad** > **Add Packages...**
 * Set the **Add Packages** window's **Source** drop-down to "nuget.org"
-* Enter NSwag.AspNetCore in the search box
-* Select the NSwag.AspNetCore package from the results pane and click **Add Package**
+* Enter "NSwag.AspNetCore" in the search box
+* Select the "NSwag.AspNetCore" package from the results pane and click **Add Package**
 
-# [Visual Studio Code](#tab/visual-studio-code)
+### [Visual Studio Code](#tab/visual-studio-code)
 
 Run the following command from the **Integrated Terminal**:
 
 ```console
-dotnet add TodoApi.NSwag.csproj package NSwag.AspNetCore
+dotnet add TodoApi.csproj package NSwag.AspNetCore
 ```
 
-# [.NET Core CLI](#tab/netcore-cli)
+### [.NET Core CLI](#tab/netcore-cli)
 
 Run the following command:
 
 ```console
-dotnet add TodoApi.NSwag.csproj package NSwag.AspNetCore
+dotnet add TodoApi.csproj package NSwag.AspNetCore
 ```
 
 ---
 
 ## Add and configure Swagger middleware
 
-Import the following namespaces in the `Info` class:
+Import the following namespaces in the `Startup` class:
 
-```csharp
-using NSwag.AspNetCore;
-using System.Reflection;
-using NJsonSchema;
-```
+[!code-csharp[](../tutorials/web-api-help-pages-using-swagger/samples/2.0/TodoApi.NSwag/Startup.cs?name=snippet_StartupConfigureImports)]
 
 In the `Startup.Configure` method, enable the middleware for serving the generated Swagger specification and the Swagger UI:
 
-[!code-cs[](../tutorials/web-api-help-pages-using-swagger/samples/TodoApi.NSwag/Startup.cs?name=snippet_Configure&highlight=4,7-10)]
+[!code-csharp[](../tutorials/web-api-help-pages-using-swagger/samples/2.0/TodoApi.NSwag/Startup.cs?name=snippet_Configure&highlight=6-10)]
 
-Launch the app. Navigate to `/swagger` to view the Swagger UI. Navigate to `/swagger/v1/swagger.json` to view the Swagger specification.
+Launch the app. Navigate to `http://localhost:<random_port>/swagger` to view the Swagger UI. Navigate to `http://localhost:<random_port>/swagger/v1/swagger.json` to view the Swagger specification.
```

Comment:
What do you think about ``http://localhost:<port>/swagger` given that CLI uses 5000. 
We should probably standardize this.

Reply:
Sorry, formatting error. It's fixed now
`http://localhost:<port>/swagger`

Reply:
I like that format and agree we should standardize on it all throughout the docset.